src/libawkward/array/RegularArray.cpp

```diff
@@ -1152,7 +1152,7 @@ namespace awkward {
       Index64 nextcarry(len);
       Index64 nextadvanced(len);
 
-      struct Error err = awkward_regulararray_getitem_next_array_advanced_64(
+      struct Error err = kernel::regulararray_getitem_next_array_advanced<int64_t>(
```

Comment:
Don't: the kernel name is `awkward_regulararray_getitem_next_array_advanced_64`.